iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift

```diff
@@ -85,22 +85,21 @@ class LoginViewController: UIViewController {
             dmePort = 38001
             appName = "ARShooter"  // ARShooter
             appVers = "1.0"
-            devName = "franklin-mobiledgex"
+            orgName = "franklin-mobiledgex"
             carrierName = "TDG"
             authToken = nil
-            location = ["longitude": -122.149349, "latitude": 37.459609]  // Get actual location and ask user for permission
+            location = MobiledgeXiOSLibrary.MatchingEngine.Loc(latitude: 37.459609, longitude: -122.149349)  // Get actual location and ask user for permission
             uniqueIDType = nil
             uniqueID = nil
             cellID = 0
             tags = nil
         } else {
             appName = matchingEngine.getAppName()
             appVers = matchingEngine.getAppVersion()
-            devName = "franklin-mobiledgex"
-            // carrierName = matchingEngine.getCarrierName() ?? "TDG"
+            orgName = "franklin-mobiledgex"
             carrierName = "TDG"
             authToken = nil
-            location = ["longitude": -122.149349, "latitude": 37.459609]  // Get actual location and ask user for permission
+            location = MobiledgeXiOSLibrary.MatchingEngine.Loc(latitude: 37.459609, longitude: -122.149349)  // Get actual location and ask user for permission
```

Comment:
Doesn't CoreLocation ask upon use? (Don't know, that's how Unity does it).

Reply:
Updated with CoreLocation to grab user location when demo == false